test/e2e/lib/components/guide-component.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * External dependencies
+ */
+import { By, Key } from 'selenium-webdriver';
+
+/**
+ * Internal dependencies
+ */
+import * as driverHelper from '../driver-helper.js';
+
+import AsyncBaseContainer from '../async-base-container.js';
+
+export default class GuideComponent extends AsyncBaseContainer {
+	constructor( driver ) {
+		super( driver, By.css( '.components-guide' ) );
+	}
+
+	async dismissModal( selector, waitOverride ) {
```

Comment:
```suggestion
	async dismiss( selector, waitOverride ) {
```

Nit: given that we know the context is a welcome guide do we need to be specific about the visual implementation? Just "dismissing" the Guide seems enough?

Reply:
True! ae3be17